e2e/atlas/atlas_e2e_test_generator_test.go

```diff
@@ -141,26 +140,26 @@ func (g *atlasE2ETestGenerator) newAvailableRegion(tier, provider string) (strin
 func (g *atlasE2ETestGenerator) getHostnameAndPort() (string, error) {
 	g.t.Helper()
 
-	process, err := g.getFirstProcess()
+	p, err := g.getFirstProcess()
 	if err != nil {
 		return "", err
 	}
 
 	// The first element may not be the created cluster but that is fine since
```

Comment:
[nit] should this comment refer to the line below or to line 143?

Reply:
I think yes, Should be on 143, I'll do a follow up to fix that